lib/common/output_text.c

```diff
@@ -145,7 +145,10 @@ text_err(pcmk__output_t *out, const char *format, ...) {
      * potentially indented list output.
      */
     len = vfprintf(stderr, format, ap);
-    pcmk__assert(len >= 0);
+    if (len < 0) {
+        fprintf(stderr, "Error when printing error message\n");
```

Comment:
If vfprintf() failed, fprintf() will too

Originally I was thinking we should crm_err() the original message for the unlikely case that someone enabled file logging in their tool environment (probably for debugging), but looking at the rest of the text implementation, we often call `fprintf()` without error-checking, so we might as well just do that consistently (here, text_info(), pcmk__formatted_vprintf())